vayesta/core/types/wf/ccsdtq.py

```diff
@@ -14,12 +14,13 @@ def CCSDTQ_WaveFunction(mo, *args, **kwargs):
 
 class RCCSDTQ_WaveFunction(wf_types.WaveFunction):
 
-    def __init__(self, mo, t1, t2, t3, t4):
+    def __init__(self, mo, t1, t2, t3, t4_abab, t4_abaa):
```

Comment:
Can we keep `t4` as it was - if it requires two spin-blocks even for RHF, we can simple store them in a tuple of size 2,
with the "abaa" block first. This will then agree with the future signature of UCCSDTQ, which will have many blocks.

Reply:
Yes - good idea. Done.